client/src/components/Tea/TeaCollectionTable/TeaCollectionTable.jsx

```diff
@@ -3,149 +3,144 @@ import { Link } from "react-router-dom";
 import { library } from "@fortawesome/fontawesome-svg-core";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faAngleDown, faAngleUp } from "@fortawesome/free-solid-svg-icons";
+import { InputField } from "../../FormComponents/InputField";
 
 library.add(faAngleDown, faAngleUp);
 
-export class TeaCollectionTable extends React.Component {
-  render() {
-    return (
-      <div className="container">
-        <div className="columns is-pulled-right">
-          <form onSubmit={this.props.handleFilterClick}>
-            <div className="field has-addons">
-              <div className="control">
-                <span className="select is-small">
-                  <select
-                    name="filterCategory"
-                    value={this.props.formControls.filterCategory}
-                    onChange={this.props.filterDropdownChangeHandler}
-                  >
-                    <option key="category" value="">
-                      Filter by
+export const TeaCollectionTable = React.memo(props => {
```

Comment:
In the future, if you already have a React.Component like this (where it will always return the same output if passed the same props/state), you can just make it a React.PureComponent to get the performance boost, rather than re-writing the whole thing as a function and wrapping it in React.memo.

https://reactjs.org/docs/react-api.html#reactpurecomponent